pkg/openstack/loadbalancer.go

```diff
@@ -1514,6 +1514,9 @@ func (lbaas *LbaasV2) checkService(service *corev1.Service, nodes []*corev1.Node
 
 	// If in the config file internal-lb=true, user is not allowed to create external service.
 	if lbaas.opts.InternalLB {
+		if !getBoolFromServiceAnnotation(service, ServiceAnnotationLoadBalancerInternal, false) {
+			klog.V(1).Infof("Annotation %q is set to false (default) but internal-lb=true, enforcing internal LB", ServiceAnnotationLoadBalancerInternal)
```

Comment:
I 'd think we should log this unconditionally if it's helpful for operator?

Reply:
I don't mind. Please provide a suggestion if you think it's better.

Reply:
yes, please if you can ,thank you~

Reply:
Would this work?

```
klog.V(1).Infof("Detected internal-lb=true setting, enforcing internal LB")
```

Is the v-level ok?

Reply:
Please use the Structured Logging and use `V(3)`:

```
klog.V(3).InfoS("Enforcing internal LB", "annotation", true, "config", false)
```

Reply:
@lingxiankong any reason for v3? I think it's a mutating and only log once, should we always log it?

Reply:
Because the occm is enforcing the load balancer configuration which is different with the user's expectation, the log here makes sense and it's just a warning, the operator still has a chance to see that when debugging. Using v3 because it doesn't fall into the default log level based on https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

Reply:
ok, well point, agree , thanks~